pkg/network/iface/link.go

```diff
@@ -203,6 +208,63 @@ func (l *Link) unsetRules4DHCP() error {
 	return nil
 }
 
+func (l *Link) EnsureIptForward() error {
+	sysctlInterface := sysctl.New()
+	isForward, err := sysctlInterface.GetSysctl(ipv4Forward)
+	if err != nil {
+		return err
+	}
+	if isForward != 1 {
+		if err := sysctlInterface.SetSysctl(ipv4Forward, 1); err != nil {
+			return err
+		}
+	}
+
+	ipt, err := iptables.NewWithProtocol(iptables.ProtocolIPv4)
+	if err != nil {
+		return err
+	}
+	rules, err := ipt.List(tableFilter, chainForward)
+	if err != nil {
+		return err
+	}
+
+	for _, rule := range rules {
+		if strings.HasPrefix(rule, "-P "+chainForward) {
+			if strings.Fields(rule)[2] == "ACCEPT" {
+				return nil
+			}
+			break
+		}
+	}
```

Comment:
Use `ipt.Exists()` for the check

Reply:
What we check is the policy, not a rule.  The function `ipt.Exists` doesn't work in this case.